src/modules/Home/components/home-entities/template.php

```diff
@@ -36,11 +36,11 @@
                         </div>
                     </div>
                     <div class="card__left--img">
-                        <img src="<?php $this->asset($app->config['module.home']['home-opportunities']) ?>" />
+                        <img :src="subsite?.files?.opportunityBanner ? subsite?.files?.opportunityBanner?.url : '<?php $this->asset($app->config['module.home']['home-opportunities']) ?>'" />
```

Comment:
Esta alteração só surgiu na v7.5.0, portanto a linha deve ser mantida inalterada conforme a tag [v7.4.26 do repositório original](https://github.com/mapasculturais/mapasculturais/blob/v7.4.26/src/modules/Home/components/home-entities/template.php). ATENÇÃO pois há neste arquivo mais 4 linhas semelhantes que não devem ser alteradas.
```suggestion
                        <img src="<?php $this->asset($app->config['module.home']['home-opportunities']) ?>" />
```